dev/percy/funnel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "data": [
+    {
+      "type": "funnel",
+      "y": ["Lead", "Pipeline", "Proposal", "Negotiation", "Closed (Won)"],
```

Comment:
just a general, more plot.js type question. Why are the axes for funnel (x/y) and funnelarea (labels/values), why couldn't they both have been named (labels/values) ? I feel like that would have been easier understand for a business type chart.

Reply:
yeah this is because `funnel` is based on `bar` and is on cartesian axes, but `funnelarea` is based on `pie` and has no axes or orientation.